<launch>
<node name="cone_bottom_detector" pkg="rr_iarrc" type="cone_bottom_detector" output="screen">
        <!--HSV color threshold for orange cones-->
        <param name="orange_low_H" type="int" value="5" />
        <param name="orange_high_H" type="int" value="15" />
        <param name="orange_low_S" type="int" value="140" />
        <param name="orange_low_V" type="int" value="140" />

        <!--The larger the height the lower it will be on the screen (Image Height = 1280)-->
        <param name="blockSky_height" type="int" value="530" />
        <param name="blockWheels_height" type="int" value="1200" />
        <param name="blockBumper_height" type="int" value="1200" />

        <param name="subscription_node" type="string" value="/camera/image_color_rect"/>
    </node>
</launch>
